documentation/server/discord/bot.md

```diff
@@ -6,49 +6,49 @@ description: Learn more about the Discord Bot
 
 ## Introduction
 
-Integrate an Discord Bot to your gamemode can have some benefits for you. Creating a whitelist, use the awesome
-permission layer from discord for ingame actions, read some server stats, kick players and so on. The list can be go
+Integrate a Discord Bot to your gamemode can have some benefits for you. Creating a whitelist, use the awesome
+permission layer from Discord itself for in-game actions, read some server stats, kick players and so on. The list can go
 further.
 
-{% hint style="info" %} Creating your own bot can not be simpler as this. {% endhint %}
+{% hint style="info" %} Creating your own bot cannot be simpler as this. {% endhint %}
 
 ## Setup the Bot
 
 ### Step 1
 
 Go to [https://discord.com/developers/applications](https://discord.com/developers/applications) and create an account
-and application it not already exists
+and application if not already existent.
 
 ### Step 2
 
-Visit the **Bot** Page and **Add Bot** or using an existing one
+Visit the Page **Bot** and **Add Bot** or use an existing one.
 
 ### Step 3
 
-Store the Bot Token inside your environment.json and setup like the picture below
+Store the Bot Token inside your `environment.json` and set it up based on the picture below.
 
 ![Uncheck Public Bot and Check Privileged Gateway and Presence Intent](../../../.gitbook/assets/discord/intents.png)
 
 ### Step 4
 
-Generate an OAuth url and setup permissions for your bot, copy / paste to your browser and invite him to your server.
+Generate an OAuth URL and setup permissions for your bot, copy / paste it to your browser and invite him to your Server.
```

Comment:
`paste it to your browser and invite him to your Server.` to ` paste it into your browser and invite it to your Server.`

Reply:
You really think `it` is better then `him`? Sounds not good to me, but i am german XD

Reply:
It works ok in English, It's my native language 😄